src/Components/ServiceScene/Breakdowns/AddToFiltersButton.test.tsx

```diff
@@ -131,7 +131,7 @@ describe('addToFilters and addAdHocFilter', () => {
       const lookupVariable = jest.spyOn(sceneGraph, 'lookupVariable').mockReturnValue(adHocVariable);
       addToFilters('indexed', 'value', 'include', {} as SceneObject);
 
-      expect(lookupVariable).toHaveBeenCalledWith(VAR_LABELS, expect.anything());
+      expect(lookupVariable).toHaveBeenCalledWith(VAR_FIELDS, expect.anything());
```

Comment:
Isn't this change rather an indication that something is wrong? 
The test says
> identifies indexed lables and uses the appropriate variable

and with the change we seem to add it to the `FIELDS` variable and not `LABELS` anymore?

Reply:
The app does create the correct query expression tho. Confusing

Reply:
Good catch, looks like we were stubbing the object wrong in the test. Dunno how that was causing it to update the wrong variable, but after updating the stub to be a DetectedLabel[], the original assertions started working.